Purchases/Purchasing/StoreKitAbstractions/StoreProduct.swift

```diff
@@ -211,3 +211,28 @@ extension StoreProduct {
     }
 
 }
+
+// MARK: - Renames
```

Comment:
I thought putting these here was better instead of `Obsoletions.swift` since these aren't our own deprecated APIs.

Reply:
great call